schema/filter_artworks.js

```diff
@@ -99,8 +99,7 @@ export const FilterArtworksType = new GraphQLObjectType({
       resolve: ({ options }) => {
         const { tag_id } = options
         if (tag_id) {
-          return gravity(`tag/${tag_id}`)
-            .then(tag => assign({ context_type: "Tag" }, tag))
+          return gravity(`tag/${tag_id}`).then(tag => assign({ context_type: "Tag" }, tag))
```

Comment:
no changes, looks like it might have got updated as I looked through the codebase